templates/core/terraform/scripts/upload_static_web.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
```

Comment:
I think this file should move out of this folder and into the devops/scripts one as it has nothing to do with Terraform and called directly from the makefile.
I know that you didn't create it so will understand if you decide to keep it here anyway.